### DISCUSSION TITLE
Laravel 11 support

### DISCUSSION THREAD

**Discussion by ntgraph** [lucidarch/lucid#56] [General]
Hi, I am wondering If I am the only who used this library.  This architecture is wonderful, suitable for large scale application. I am not proficient to modify/contribute/revive this library, but I really wish to use it with Laravel 11.  Anyone kind enough to update/maintain this library for compatiblity purpose with Laravel 11, even without new features?  Or is there any equivalent coding practice that I miss out, then please guide.  Thanks in advance

**Comment by Mulkave** [comment]
Coming soon! Sorry for the long silence. I already posted in a the other discussion that you're part of https://github.com/lucidarch/lucid/discussions/54

**Reply by ntgraph** [reply]
Hi, this is good news to hear you are back. Thank you.
